js/arena.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+import Gladiator from './gladiator.js';
+export default class Arena{
+    constructor(name , gladiators = []){
+        const cName = name.charAt(0).toUpperCase() + name.slice(1);
+        this.name = cName;
+        this.gladiators = gladiators;
+    }
+    addGlaidiator(gladiator){
+        if(this.gladiators.length < 2){
+        this.gladiators.push(gladiator);
+        }
```

Comment:
an else condition should be added to tell the user that there are more than 2 gladiators in the arena. Should look something like this:
  else {
            console.log(`Can't have more than 2 fighters in the Arena`)
        }